sale_planner_calendar/models/calendar_event.py

```diff
@@ -87,6 +87,15 @@ class CalendarEvent(models.Model):
     sanitized_partner_mobile = fields.Char(compute="_compute_sanitized_partner_mobile")
     location_url = fields.Char(compute="_compute_location_url")
     categ_icons = fields.Char(compute="_compute_categ_icons")
+    sale_planner_rating = fields.Selection(
+        [
+            ("1", "Bad"),
+            ("3", "Fair"),
+            ("5", "Good"),
+        ],
+        required=True,
+    )
+    comment_rating = fields.Text()
```

Comment:
For the moment we can use the existing comment field so as not to overload if not necessary.